controller/healthsyncer/healthsyncer.go

```diff
@@ -0,0 +1,133 @@
+package healthsyncer
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	client "github.com/rancher/cluster-agent/client"
+	"github.com/rancher/cluster-agent/controller"
+	clusterv1 "github.com/rancher/types/apis/cluster.cattle.io/v1"
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	syncInterval                    = 15 * time.Second
```

Comment:
do we want to make this value configurable?

Reply:
Since I am not watching componentstatuses anymore, I checked the interval k8s refreshes these components in its probe. Making it configurable would mean probably missing out on update?